feature_engine/transformation/log.py

```diff
@@ -147,6 +149,45 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
             X.loc[:, self.variables_] = np.log10(X.loc[:, self.variables_])
 
         return X
+    
+    def inverse_transform(self, X: pd.DataFrame) -> pd.DataFrame:
+        """
+        Transforms the variables using exp transformation.
+
+        Parameters
+        ----------
+        X: Pandas DataFrame of shape = [n_samples, n_features]
+            The data to be transformed.
+
+        Raises
+        ------
+        TypeError
+            If the input is not a Pandas DataFrame
+        ValueError
+            - If the variable(s) contain null values
+            - If the df has different number of features than the df used in fit()
+            - If some variables contains zero or negative values
+
+        Returns
+        -------
+        X: pandas dataframe
+            The dataframe with the transformed variables.
+        """
+
+        # check input dataframe and if class was fitted
+        X = super().transform(X)
+
+        # check contains zero or negative values
+        if (X[self.variables_] <= 0).any().any():
+            raise ValueError(
+                "Some variables contain zero or negative values, can't apply log"
+            )
+
+        # inverse_transform
+        X.loc[:, self.variables_] = np.exp(X.loc[:, self.variables_])
```

Comment:
this function reverses log natural (base e). But what if user used log10? Can you add this?

Reply:
I feel like we discussed about this one in Telegram. There is not a function to inverse log10.

Do you think something like this will work?

elif self.base == "10":
            X.loc[:, self.variables_] = np.array(10 ** X.loc[:, self.variables_])

Reply:
Test case is passed, so looks like it should ...

Reply:
yes, that works. Also np.exp(10, variable) should work for variable in self.variables_